index

```diff
@@ -12,7 +12,7 @@ args, _ = parser.parse_known_args()
 # Iterate over the [name]=[path] pairs
 for collection in args.json["collections"]:
     name, path = collection["name"], collection["path"]
-    subprocess.check_call("zcat $(find {1} -path *disk4/cr/* -type f -prune -o -name '*.z') | parse_collection -f trectext "
+    subprocess.check_call("zcat $(find {1} -type f  -name '*.*z' \( -path '*/disk4/fr94/[0-9]*/*' -o  -path '*/disk4/ft/ft*' -o -path '*/disk5/fbis/fb*' -o -path '*/disk5/latimes/la*' \) -type f) | parse_collection -f trectext "
```

Comment:
`-type f` is repeated, is that intentional or a mistake?

Reply:
Nice catch! It was a mistake.
I have tested the command and checked all the produced file.